build

```diff
@@ -11,14 +11,22 @@ METEOR_DIR=$SCRIPTPATH/meteor_core
 
 [ -f $METEOR_DIR/client/loadClientBundle.html ] && rm $METEOR_DIR/client/loadClientBundle.html
 
-SERVER_BUNDLE="assets/server.bundle.js"
-CLIENT_BUNDLE="assets/client.bundle.js"
-
 cd $WEBPACK_DIR
-env NODE_ENV=production $SCRIPTPATH/node_modules/webpack/bin/webpack.js --config webpack.config.client.prod.js --progress --colors
-env NODE_ENV=production $SCRIPTPATH/node_modules/webpack/bin/webpack.js --config webpack.config.server.prod.js --progress --colors
-cd $METEOR_DIR
-ln -sf $WEBPACK_DIR/$SERVER_BUNDLE server/server.bundle.js
-ln -sf $WEBPACK_DIR/$CLIENT_BUNDLE client/client.bundle.js
-rm -rvf ../dist
-METEOR_SETTINGS=$(cat ../settings/prod.json) && meteor build ../dist
+rm -rvf assets
+(
+  env NODE_ENV=production ../node_modules/webpack/bin/webpack.js --config webpack.config.client.prod.js --progress --colors
+  env NODE_ENV=production  ../node_modules/webpack/bin/webpack.js --config webpack.config.server.prod.js --progress --colors
+
+  # wait for bundles to be created
+  SERVER_BUNDLE="assets/server.bundle.js"
+  CLIENT_BUNDLE="assets/client.bundle.js"
+  (while : ; do
+    [ -f "$SERVER_BUNDLE" ] && [ -f "$CLIENT_BUNDLE" ] && break
+    sleep 1
+  done) &&
+
```

Comment:
This is unnecessary for the build script because the webpack calls above are blocking, and the bundles will exist when they finish.  This wait loop was only necessary when running the webpacks in watch mode in separate processes